openprescribing/pipeline/management/commands/create_importlog_for_measures.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management import BaseCommand
+from frontend.models import ImportLog
+
+
+class Command(BaseCommand):
+    help = "Records the fact that measures have finished importing"
+
+    def handle(self, *args, **kwargs):
+        date = ImportLog.objects.latest_in_category("prescribing").current_at
+        ImportLog.objects.get_or_create(category="measures", current_at=date)
```

Comment:
Why can't this happen at the end of `import_measures`?

Reply:
The main reason is the one I discussed above i.e. we need PPU to finish processing as well. But apart from that we'd also need to make sure that it only create the ImportLog when it finished importing all measures, not when we run it on an ad hoc basis to import a single measure. And that would involve more messy logic which I didn't relish.

Reply:
Got it.